internal/internal_workflow_testsuite.go

```diff
@@ -1208,11 +1210,13 @@ func (env *testWorkflowEnvironmentImpl) RequestCancelWorkflow(domainName, workfl
 		childHandle.handled = true
 		childEnv := childHandle.env
 		childEnv.cancelWorkflow()
+		callback(nil, nil)
```

Comment:
this should be handled within childEnv.cancelWorkflow()

Reply:
same